classes/Cart.php

```diff
@@ -1712,7 +1712,7 @@ public function deleteProduct(
                 WHERE `id_cart` = ' . (int) $this->id . '
                 AND `id_product` = ' . (int) $id_product . '
                 AND `id_customization` = ' . (int) $id_customization .
-                ($id_product_attribute != null ? ' AND `id_product_attribute` = ' . (int) $id_product_attribute : '')
```

Comment:
No need this change since `0 == null`

```php
var_dump(null == 0);  => bool(true)
```

Reply:
Should we change to this ? : 

$id_product_attribute !== 0

Reply:
I'm more in favor of
```suggestion
                ($id_product_attribute !== 0 ? ' AND `id_product_attribute` = ' . (int) $id_product_attribute : '')
```

But not sure it will change something for the two affected requests